.claude/settings.local.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
I think this needs to be named `settings.json` if its intended to share with the rest of the team (`settings.local.json` is just _your_ project specific settings). I'd also remove some of the allowed commands for writing (`chmod`, `mkdir`) as this should be developer preference. We probs don't need the job backfill commands either